### ISSUE TITLE
create a PGN of the game, after finished

### ISSUE THREAD

**Issue by micha003** [trevorbayless/cli-chess#19]
Hi, I think, it should be possible to recieve a PGN-file after a played game is finished to analyze it further.
To write a PGN, you can use python-chess by niklasf. 

For example: you play a game against the Fairy Stockfish and you want to analyze it on Lichess or Chessx with Stockfish. Therefore you need a PGN, to insert it and let Stockfish analyze it.

**Comment by trevorbayless** [comment]
Yes, this is absolutely possible using `python-chess` and a feature that I also believe would add great value for offline games. Thank you for the suggestion!
